Data/Versions.hs

```diff
@@ -615,58 +615,58 @@ epoch f v = fmap (\ve -> v { _vEpoch = ve }) (f $ _vEpoch v)
 -- Not guaranteed to have well-defined ordering (@Ord@) behaviour, but so far
 -- internal tests show consistency. `messMajor`, etc., are used internally where
 -- appropriate to enhance accuracy.
-data Mess = VLeaf [T.Text] | VNode [T.Text] VSep Mess
+data Mess = Mess (NonEmpty T.Text) (Maybe (VSep, Mess))
```

Comment:
The node of a `Mess` can no longer be empty. Further, this has been changed from a sum type to a record.